addon/services/raven.js

```diff
@@ -100,12 +95,14 @@ export default Service.extend({
         environment,
         includePaths,
         whitelistUrls,
-        release: this.get(serviceReleaseProperty) || config.APP.version
+        release: get(this, serviceReleaseProperty) || config.APP.version
       }, ravenOptions);
 
       Raven.config(dsn, ravenConfig);
     } catch (e) {
-      Ember.Logger.warn('Error during `sentry` initialization: ' + e);
+      warn('Error during `sentry` initialization: ' + e, {
+        id: 'ember-cli-sentry.initialization-error'
+      });
```

Comment:
adding an `id` here seems unrelated to the PR goal of moving towards RFC176 imports